integration/nwo/token/orion/template.go

```diff
@@ -37,6 +37,9 @@ token:
       auditdb:
         persistence:
           type: unity
+      tokenlockdb:
```

Comment:
This should not be needed. The driver should switch to the shared configuration under `db`, no?

Reply:
For Orion it was needed. It failed otherwise and the other db's also have it.

Reply:
so, why doesn't the fabric template need it?

Reply:
Because you take the following configs in this order:
```
dbconfig.NewConfig(configProvider, "tokenlockdb.persistence.type", "db.persistence.type"))
```
For Orion, the latter is not defined, for Fabric it is.